src/Microsoft.Tye.Hosting/Model/ProjectRunInfo.cs

```diff
@@ -24,6 +24,8 @@ public ProjectRunInfo(ProjectServiceBuilder project)
             RunCommand = project.RunCommand;
             RunArguments = project.RunArguments;
             PublishOutputPath = project.PublishDir;
+            ContainerBaseImage = project.ContainerInfo!.BaseImageName;
```

Comment:
I think this should work:
```suggestion
            ContainerBaseImage = project.ContainerInfo?.BaseImageName;
```